src/index.js

```diff
@@ -1,9 +1,29 @@
 /* global global, window, module */
+const { v4 } = require("uuid");
 const { sha3_512: sha3 } = require("@noble/hashes/sha3");
 
 const defaultLength = 24;
 const bigLength = 32;
 
+const randomUUID = v4;
+
+const typedArrayToString = (arr) => arr.map((x) => `${x}`).join("");
+
+const hexToScaled = (str, max = 0xffffffff) => Number(`0x${str}`) / max;
+
+// It's not a requirement to use a UUID. It's just a convenient
+// hack in JavaScript to get CSPRNG values if they're available. We never use
+// the second half, so we only convert the first half.
+const uuidToNumbers = (str) => {
+  const string = str.replace(/-/g, "");
+  const a = hexToScaled(string.slice(0, 8));
```

Comment:
If we're going to use this, we need to cut out the fixed bits from the UUID. It may be better to use a getRandomBytes function after all, but we need to come up with a solution that will work on all our target platforms.